core/primitives/src/serialize.rs

```diff
@@ -78,34 +43,6 @@ pub trait BaseDecode: for<'a> TryFrom<&'a [u8], Error = Box<dyn std::error::Erro
     }
 }
 
-macro_rules! pretty_bytes_serialization {
```

Comment:
I decided to be more explicit and use `_base64` suffix on the fields serialized with base64 encoding, which requires serde renaming anyway, so I rolled back this macro implementation and replaced it with appropriate field attributes.